python/unitytrainers/models.py

```diff
@@ -85,6 +82,15 @@ def create_vector_input(self, name='vector_observation'):
             self.vector_in = tf.placeholder(shape=[None, 1], dtype=tf.int32, name='vector_observation')
             return self.vector_in
 
+    def create_normalizer_update(self):
```

Comment:
I would rather have this method return `update_mean` and `self.update_variance` and take in `self.vector_in` as input.

Reply:
Done.